lib/statsd/instrument/rubocop/positional_arguments.rb

```diff
@@ -0,0 +1,59 @@
+# frozen-string-literal: true
+
+module RuboCop
+  module Cop
+    module StatsD
+      class PositionalArguments < Cop
+        MSG = 'Use keyword arguments for StatsD calls'
+
+        STATSD_SINGLETON_METHODS = %i{increment gauge measure set histogram distribution key_value}
+
+        def on_send(node)
+          if node.receiver&.type == :const && node.receiver&.const_name == "StatsD"
+            if STATSD_SINGLETON_METHODS.include?(node.method_name)
+              arguments = node.arguments
+              if arguments.length > 2 && ![:block_pass, :hash].include?(arguments[2].type)
```

Comment:
Should `[:block_pass, :hash]` also be moved to a constant to avoid allocation each time this line is run?